cmd/relay-gossip/main.go

```diff
@@ -213,22 +219,25 @@ var clientCmd = &cli.Command{
 			return xerrors.Errorf("constructing host: %w", err)
 		}
 
-		c, err := client.NewWithPubsub(ps, cctx.String("network-name"))
-		if err != nil {
-			return xerrors.Errorf("constructing client: %w", err)
+		// TODO extract group from CLI args
+		var group *key.Group
```

Comment:
it'll be hard to pass in fully on CLI. if passed in, probably more natural to pass in as a json file.

Reply:
We have to/from TOML that operate on `key.Group`s. Should good?